omod/src/main/resources/config.xml

```diff
@@ -265,6 +265,14 @@
         </description>
     </globalProperty>
 
+    <globalProperty>
+        <property>coreapps.conditionListClasses</property>
+        <defaultValue>8d4918b0-c2cc-11de-8d13-0010c6dffd0f</defaultValue>
```

Comment:
Didn't we have more than one for the default?

Reply:
it takes just Diagnosis as default with uuid: 8d4918b0-c2cc-11de-8d13-0010c6dffd0f